examples/gno.land/r/x/grc20_dynamic_call/registry/registry.gno

```diff
@@ -0,0 +1,101 @@
+package registry
+
+import (
+	"std"
+
+	"gno.land/r/demo/users"
+)
+
+const APPROVED_UNREGISTER_CALLER = "g1sqt92sa06ugh8nlt98kyghw83qy84paf4csyh6"
+
+var registered = []GRC20Pair{}
```

Comment:
Have you thought about using a map for these registered pairs (or `iavl`)? You could map `pkgPath -> GRC20Pair` I think in this way.

Your lookups would be much quicker

Reply:
Aha, good point. Personally I prefer `map`, will change after meeting